testing/android-test-configurations/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Toast Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+android {
+    compileSdkVersion(29)
+
+    sourceSets["test"].java.srcDir("../android/src/test/java")
+}
+
+localProtokt()
+pureKotlin()
+
+dependencies {
+    testImplementation(libraries.protobufLite)
```

Comment:
Weird things happen if you try to use `protobuf` or `testProtobuf` configurations here. I can't tell if it's our fault or if it's something upstream.

```
Execution failed for task ':testing:android:generateDebugProto'.
> protoc: stdout: . stderr: /<...>/protokt/testing/android/build/extracted-protos/main/google/protobuf/any.proto: Input is shadowed in the --proto_path by "/<...>/protokt/testing/android/build/extracted-include-protos/debug/google/protobuf/any.proto".  Either use the latter file as your input or reorder the --proto_path so that the former file's location comes first.
```